packages/twig/src/patterns/components/icon/icon.wingsuit.yml

```diff
@@ -5,7 +5,7 @@ icon:
   description: A component for displaying an svg icon.
   settings:
     name:
-      type: string
+      type: value
```

Comment:
What is the difference between type string and type value?

Reply:
On the Drupal site, those are mapped to Drupal plugin IDs, and the plugin `string` does not exist, while `value` does.